lib/req-res/requester.js

```diff
@@ -23,6 +23,13 @@ function Requester ( options ) {
 
 	if ( data instanceof Error ) {
 		this.error = data;
+	} else {
+		this.queue = mixins.getQueueName( options.messageType, 'req-res' );
+
+		// set error listener
+		this.on( 'error', ( errorReq ) => {
+			config.logger.error( `Queue: ${this.queue} -  ${errorReq.message}` );
```

Comment:
I forgot that I have create a logger that will toggle between `console.log` and `sinet/log` or what ever log is passed. I'm using it instead. 

https://github.com/sinet/lapin/pull/104/files#diff-adc440c0d233640bbb310d9309de968cR24